src/pages/User/components/UserData/UserData.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div class="UserInformation">
+    <div class="UserInformation__text">
+      <span>Nombres</span>
+      <p>{{ firstName }}</p>
+    </div>
+    <div class="UserInformation__text">
+      <span>Apellidos</span>
+      <p>{{ lastName }}</p>
+    </div>
+    <div class="UserInformation__text">
+      <span>codigo</span>
+      <p>{{ memberCode }}</p>
+    </div>
+    <div class="UserInformation__password">
+      <div>
+        <span>Contraseña</span>
+        <p>************</p>
+      </div>
+      <div>
+        <q-btn unelevated rounded label="Cambiar Contraseña" />
+      </div>
+    </div>
+    <div class="UserInformation__text">
+      <span>correo</span>
```

Comment:
"correo" should not be "Correo"?

```suggestion
      <span>Correo</span>
```